### DISCUSSION TITLE
2023년 05월 12일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#53] [General]
# 2023년 05월 12일

[ 2023-05-01 ] [ 2023-05-02 ] [ 2023-05-04 ] [ 2023-05-07 ] [ 2023-05-08 ] [ 2023-05-10 ] [ 2023-05-11 ] [ 2023-05-12 ]

http://vero.wiki/2023-05-12/

**Comment by drunkenhw** [comment]
자제하는 자세 아주 멋지네요~!

**Reply by Cyma-s** [reply]
박스터의 칭찬은 귀한데요?
